astropy/units/equivalencies.py

```diff
@@ -687,3 +687,32 @@ def plate_scale(platescale):
                          "distance/angle")
 
     return [(si.m, si.radian, lambda d: d*platescale_val, lambda rad: rad/platescale_val)]
+
+
+def littleh_as(cosmologyorH0=None):
+    """
+    Convert between quantities with little-h and the equivalent physical units.
+
+    Parameters
+    ----------
+    cosmologyorH0 : None, `~astropy.cosmology.FLRW`, or `~astropy.units.Quantity`
+        The source of the value of the Hubble constant to assume. If a cosmology
+        object is given, it will be taken from the ``H0`` attribute, or if a
+        quantity, will assume the quantity *is* ``H0``.  If None (default), use
```

Comment:
Single backticks around ``None``

Reply:
Hey I did a suggestion thingy! 🎉

Reply:
I wish there was a way to accept these without it triggering CI! I'll accept this one as a 👍 to using the feature, but the other ones I'll integrate separately :wink: